docs/source/API/core/policies/RangePolicy.rst

```diff
@@ -11,60 +11,66 @@ Usage
 
 .. code-block:: cppkokkos
 
-    Kokkos::RangePolicy<>(begin, end, args...)
-    Kokkos::RangePolicy<ARGS>(begin, end, args...)
-    Kokkos::RangePolicy<>(Space(), begin, end, args...)
-    Kokkos::RangePolicy<ARGS>(Space(), begin, end, args...)
+    Kokkos::RangePolicy<...>(begin, end)
+    Kokkos::RangePolicy<...>(begin, end, ChunkSize(cs))
+    Kokkos::RangePolicy<...>(Space(), begin, end)
+    Kokkos::RangePolicy<...>(Space(), begin, end, ChunkSize(cs))
 
-RangePolicy defines an execution policy for a 1D iteration space starting at begin and going to end with an open interval.
+RangePolicy defines an execution policy for a 1D iteration space starting at ``begin`` and going to ``end`` with an open interval.
 
 Synopsis
 --------
```

Comment:
Again, please make this entry conform to standard set in PR#385, with a 2 - 3 sentence description (instead of the Synopsis) after "Name," "Header" sections.